XIV on Mac/de.lproj/Main.strings

```diff
@@ -117,16 +117,16 @@
 "9gQ-FA-62f.title" = "D2013.06.18.0000.0000";
 
 /* Class = "NSMenuItem"; title = "Substitutions"; ObjectID = "9ic-FL-obx"; */
-"9ic-FL-obx.title" = "Substitutions";
+"9ic-FL-obx.title" = "Ersetzungen";
 
 /* Class = "NSButtonCell"; title = "Quit XIV on Mac upon game exit"; ObjectID = "9ie-LK-Dur"; */
-"9ie-LK-Dur.title" = "Quit XIV on Mac upon game exit";
+"9ie-LK-Dur.title" = "XIV on Mac beim Beenden des Spiels beenden";
```

Comment:
Maybe use

> *nach Beenden des Spiels schließen*

Your translation is OK but it uses word repetition again (Beenden ... beenden)